io.openems.edge.timeofusetariff.entsoe/test/io/openems/edge/timeofusetariff/entsoe/TouEntsoeTest.java

```diff
@@ -135,13 +136,13 @@ private static ZonedDateTime toZonedDateTime(int year, int month, int day, int h
 
 	@Test
 	public void testStandardTariffOnJuly15At2PM() throws OpenemsNamedException {
-		var helper = this.buildHelper("BAYERNWERK", "2025-07-15T00:00:00Z");
-		final var testTime = toZonedDateTime(2025, 7, 15, 14, 0);
+		var helper = this.buildHelper("BAYERNWERK", "2026-07-15T00:00:00Z");
```

Comment:
Why not use YEAR here in "testStandardTariffOnJuly15At2PM, testLowTariffOnOctober20At3AM, testHighTariffOnOctober20At5PM" as well?